### PR TITLE
Resolved Messages

### LINKED ISSUE #295: Mark messages as resolved
A lecturer should be able to mark a message as resolved. One could then hide resolved messages to better follow the current discussion.

### PR DESCRIPTION
**Pull request by MatthiasReumann**
- Closes #295
- Make messages resolvable
- Made Admin buttons more verbose
- Added custom tooltip utilities (Not in use right now)

Should messages be un-resolvable too?

image

### CODE REVIEW

File: web/template/partial/stream/chat/tooltip-bottom.gohtml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{{define "tooltip-bottom"}}
```
Comment (joschahenningsen) [inline web/template/partial/stream/chat/tooltip-bottom.gohtml:1]: Some tooltip components are super neat, I can't find where they are used though :thinking:

Comment (MatthiasReumann) [reply web/template/partial/stream/chat/tooltip-bottom.gohtml:1]: Thats because they aren't used right now 😅

Comment (joschahenningsen) [reply web/template/partial/stream/chat/tooltip-bottom.gohtml:1]: Cool, hope I remember them the next time I need them :D

**Review by joschahenningsen — APPROVED** [review]
Looks great!

**Pull request merged**